src/sort.js

```diff
@@ -0,0 +1,13 @@
+import CONST from './constants';
```

Comment:
I'm not a big fan of the name of the file. `sort.js` is quite generic. Maybe it could be named something like `hot-score.js` to tell what it is about